src/Catel.MVVM/Catel.MVVM.Shared/MVVM/ViewModels/ViewModelManager.cs

```diff
@@ -412,6 +432,17 @@ private void UnregisterViewModelInstanceInternal(IViewModel viewModel)
         }
 
         /// <summary>
+        /// Gets the active view models.
+        /// </summary>
+        /// <param name="managedViewModels">Dictionary of view-models</param>
+        /// <returns></returns>
+        private static IEnumerable<IViewModel> GetAllViewModels(Dictionary<Type, ManagedViewModel> managedViewModels)
+        {
+            // TODO: refactor to internal extension method if necessary
```

Comment:
AllViewModels have been used in some other class. I have removed it by adding one of methods (I don't remember which one). I can remove this comment if you want.